client/src/main/scala/client/BackPressuredWebSocketClient.scala

```diff
@@ -78,14 +87,14 @@ final class BackPressuredWebSocketClient private (url: String)
         scheduler.reportFailure(ex)
         // retry connection in a couple of secs
         self.delaySubscription(3.seconds)
-          .onSubscribe(subscriber)
+          .subscribe(subscriber)
```

Comment:
Here we should call `unsafeSubscribeFn` because we know what we are doing (i.e. our stuff is protected from errors with try/catch, we do know how to apply back-pressure, etc). Plain users are the ones that are supposed to call `subscribe`, as it wraps `Subscriber` implementations into a `SafeSubscriber` that does error handling and applies back-pressure on `onComplete` and `onError`.